### DISCUSSION TITLE
Queries regarding GSOC projects

### DISCUSSION THREAD

**Discussion by nataliyah123** [MDAnalysis/mdanalysis#4501] [GSoC Q&A] [answered]
I am interested in two of the projects 1. extend interoperability and 2. 2D representation for rdkit

1. Can I apply to both of these projects in a separate form or in a single form.
2.  I am interested in pytraj interoperability especially in from md.universe to randomize_ion and is_solvent. so my question is, these are not analysis but the preliminary steps to analysis. is it ok to go ahead with it or you guys are looking for analysis only.  
3. pt.Trajectory(in-memory) and universe.trajectory have different functionality like multiframe and single frame or timestep. should this also be considered or not.

**Comment by jennaswa** [comment]
Hi @nataliyah123 - We do not recommend taking on more than one project (see the [GSoC with MDAnalysis FAQ](https://github.com/MDAnalysis/mdanalysis/wiki/GSoC-FAQ#can-i-do-multiple-projects) for details). I would suggest you focus on the project you are most interested in, but if you envision overlap between the 2 projects and think it is feasible to achieve during the summer, please feel free to outline your objectives in detail in your pre-proposal.

I will let the interoperability mentors (@hmacdope , @yuxuanzhuang , @xhgchen ) respond to your other questions.

**Comment by hmacdope** [comment, marked as answer]
Hi @nataliyah123 

1. You should apply to only one project. 
2. If properly considering how to do pytraj interoperability, it should be easy to use **any** pytraj method, you should have a thing about the best way to achieve this. 
3. Yes this sounds like something you should consider in your project!

**Comment by orbeckst** [comment]
@nataliyah123 if one of the replies answered the question then please mark it as "answer" — that makes the discussion very useful for others as well. But if you still have questions, please ask!
